.github/ISSUE_TEMPLATE/1-bug.yml

```
@@ -0,0 +1,11 @@
+name: Bug Report
+description: An unexpected problem or unintended behavior
+body:
+  - type: textarea
+    attributes:
+      label: Details
+      placeholder: |
+        Type your description here...
```

Nit, any reason not to do this?

```suggestion
      placeholder: Type your description here...
```